packages/react-composites/src/composites/CallComposite/pages/ConfigurationPage.tsx

```diff
@@ -340,15 +343,15 @@ export const ConfigurationPage = (props: ConfigurationPageProps): JSX.Element =>
             horizontal={!mobileWithPreview}
             horizontalAlign={mobileWithPreview ? 'stretch' : 'center'}
             verticalFill={mobileWithPreview}
-            tokens={mobileWithPreview ? configurationStackTokensMobile : undefined}
+            tokens={configurationStackTokensMobile}
           >
             {localPreviewTrampoline(
               mobileWithPreview,
               /* @conditional-compile-remove(rooms) */ !!(role === 'Consumer')
             )}
-            <Stack className={mobileView ? undefined : selectionContainerStyle}>
```

Comment:
are these tokens just always used because there is now a new style on line 354 that changes some of the spacing?

Reply:
Before they were just applied to mobile because the start button was inside the stack with the dropdowns. Now the start call button exists outside that stack it uses the same token spacing and mobile to create that space between the start call button and the content above it

Reply:
I think.. there's so many stacks floating around here, open to feedback if there's a nicer way to structure this haha